### PR TITLE
Consistent Capitalization and Spelling Issues - 2nd Batch

### PR DESCRIPTION
**Pull request by digeorgi**
Fixing the following issues:
- #184
- #177
- #176 
- #131

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -69,11 +69,11 @@ see the [tags on this repository](https://github.com/eu-digital-identity-wallet/
 
 See the list of [contributors](https://github.com/eu-digital-identity-wallet/architecture-and-reference-framework/graphs/contributors) who participated in this project.
 
-## License
+## Licence
 
 This project is licensed under the [Attribution 4.0
 International](http://creativecommons.org/licenses/by/4.0/) - see the
-[LICENSE](https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/blob/main/LICENSE) file for details.
+[LICENCE](https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/blob/main/LICENCE) file for details.
```
Comment (skounis) [inline docs/index.md:76]: This update introducing a broken link and a 404 error. We if we keep this change we should also change the file name in the target repository 

image

**Comment by skounis** [comment]
I didn't notice in this PR any changes related to #177

**Review by skounis — CHANGES_REQUESTED** [review]
1. Fix or schedule to fix the [broken link this PR introduces](https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/pull/307/files#r1745030533)
2. Include changes relevant to #177 and #131 or remove these two reference from the description of this PR

cc @duviner

File: docs/index.md (hunk 1 above)
Comment (digeorgi) [reply docs/index.md:76]: The Licence file name has been updated from LICENSE -> LICENCE

**Review by skounis — APPROVED** [review]

**Review by pinamiranda — APPROVED** [review]
Reviewed

**Pull request merged**